src/View/PublicThemes.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace SilverStripe\View;
+
+class PublicThemes implements ThemeList
+{
+    public function getThemes()
+    {
+        return PUBLIC_DIR ? ['/' . PUBLIC_DIR] : [];
+    }
```

Comment:
So simple, that’s awesome

Reply:
Could use `Path` here to be consistent. :)

Reply:
@unclecheese how do you mean? Absolute path themes in config always use `/` prefix.